app/styles/modules/_user_page.scss

```diff
@@ -15,48 +15,50 @@
 
     &.right {
       float: right;
-      margin-left: 20px;
+      margin-left: $double-gap;
     }
 
     &.left {
       float: left;
       margin-left: 0;
-      margin-right: 20px;
+      margin-right: $double-gap;
     }
   }
 
-  .text {
-    .provider {
-      margin-bottom: 10px;
+  .provider {
+    margin-bottom: $default-gap;
 
-      .favicon {
-        height: 16px;
-        margin-right: 5px;
-        vertical-align: -3px;
-        width: 16px;
-      }
+    .favicon {
+      height: $half-grid;
+      margin-right: 5px;
+      vertical-align: -3px;
+      width: 1$half-grid;
```

Comment:
@johngruen Can you confirm this weird looking `width` value here?

Should this be `$half-grid` or was the `1` in there performing some sort of magic?